flake.nix

```diff
@@ -0,0 +1,72 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+
+        # Read version from Cargo.toml
+        cargoToml = builtins.fromTOML (builtins.readFile ./crates/tattoy/Cargo.toml);
+        version = cargoToml.package.version;
+
+        nativeBuildInputs = with pkgs; [
+          rustc
+          cargo
+          pkg-config
+
+          # For tests
+          bash
+          nano
+          procps
```

Comment:
I think we could move the test-only dependencies to `checkInputs` to reduce the package closure

Reply:
Done.